app/assets/js/controllers/basket.js

```diff
@@ -0,0 +1,23 @@
+function BasketController ($scope, basketService, universal_variable, debounceOptions) {
+	var vm = this;
+
+	vm.debounceOptions = debounceOptions;
+
+	vm.updateLineItem = function(line_item, index) {
+		universal_variable.basket.line_items[index].quantity = line_item.quantity;
```

Comment:
Is there a reason to use snake case for the universal_variable service? Something to do with sending it up to the server?